### ISSUE TITLE
TabView appears to not be implementing the Selection pattern

### ISSUE THREAD

**Issue by YuliKl** [microsoft/microsoft-ui-xaml#2847]
I copied two TabView samples out of the Xaml Controls Gallery. Accessibility Insights for Windows identified this as a problem.

The following accessibility issue needs investigation.

**App:** WinUI_2-4_Test

**Element path:** tab ''

**Issue Details:** An element of the given ControlType must support the Selection pattern. [Section 508 502.3.10](https://www.access-board.gov/guidelines-and-standards/communications-and-it/about-the-ict-refresh/final-rule/single-file-version#502-interoperability-assistive-technology)

**How To Fix:** 1. Make sure the element has the appropriate ControlType property for its function.
2. If the current ControlType is correct, modify the element to support the Selection pattern.

This accessibility issue was found using Accessibility Insights for Windows, a tool that helps debug and find accessibility issues earlier. Get more information and download this tool at https://aka.ms/AccessibilityInsights.

**Comment by StephenLPeters** [comment]
@teaP  FYI

**Comment by chingucoding** [comment]
@teaP I assume you will be looking into this issue? Or would you like me to/would you be fine with me looking into this? @ranjeshj FYI

**Comment by ranjeshj** [comment]
@chingucoding go for it. TabView uses ListView under the covers which should be implementing the selection pattern. Perhaps there is some wire up missing.

**Comment by teaP** [comment]
The list inside tabview (the tab strip, if you will) has the selection pattern. I'm not certain if that is good enough or if there is some other adjustment that should be made.
